prod/westeurope/internal/api/functions_iopayportal/function_app/terragrunt.hcl

```
@@ -0,0 +1,98 @@
+dependency "subnet" {
+  config_path = "../subnet"
+}
+
+# External
+dependency "app_service_pagopaproxyprod" {
+  config_path = "../../../../external/pagopaproxyprod/app_service"
+}
+
+# Internal
+dependency "resource_group" {
+  config_path = "../../../resource_group"
+}
+
+dependency "subnet_apimapi" {
+  config_path = "../../../api/apim/subnet"
+}
+
+# Common
+dependency "virtual_network" {
+  config_path = "../../../../common/virtual_network"
+}
+
+dependency "application_insights" {
+  config_path = "../../../../common/application_insights"
+}
+
+dependency "key_vault" {
+  config_path = "../../../../common/key_vault"
+}
+
+# Include all settings from the root terragrunt.hcl file
+include {
+  path = find_in_parent_folders()
+}
+
+terraform {
+  source = "git::git@github.com:pagopa/io-infrastructure-modules-new.git//azurerm_function_app?ref=v2.1.10"
+}
+
+inputs = {
+  name                = "iopayportalapi"
```

```suggestion
  name                = "iopayportal"
```
if is possible to have same folder name